databases/better_life/better_life.rb

```diff
@@ -60,21 +60,21 @@ def create_mindful_day(db, mindful_meditation, city,state, word)
       puts Rainbow("*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*").blink.blue
         med_number.times do
         create_mindful_day(db, mindful_meditation.sample, Faker::Address.city, Faker::Address.state, Faker::Hipster.word )
-        end
+    end
+
+    better_life = db.execute("SELECT * from better_life")
+    better_life.each do |suggestion|
```

Comment:
can you work on encapsulating a lot of this logic into methods and/or classes?  It is important to start practicing the styles we are teaching for design best practices.